packages/fiori/src/FlexibleColumnLayout.ts

```diff
@@ -654,14 +663,39 @@ class FlexibleColumnLayout extends UI5Element {
 		return columnLayoutToAdjust;
 	}
 
-	async _onkeydown(e: KeyboardEvent) {
+	_onArrowKeydown(e: KeyboardEvent) {
+		if (isEnter(e) || isSpace(e)) {
+		  e.preventDefault();
+		  const focusedElement = e.target as HTMLElement;
+		  if (focusedElement === this.startArrowDOM) {
+				this.switchLayoutOnArrowPress();
+		  }
+		}
+	  }
+
+	async _onSeparatorKeydown(e: KeyboardEvent) {
+		const separator = e.target as HTMLElement;
+		if (!separator.classList.contains("ui5-fcl-separator")) {
+			return;
+		}
 		const stepSize = 2,
 			bigStepSize = this._width,
 			isRTL = this.effectiveDir === "rtl";
 		let step = 0;
+
+		if (!this.startColumnVisible && e.target === this.startSeparatorDOM && isLeft(e)) {
```

Comment:
all other additions to `_onSeparatorKeydown` are redundant in my testing because
- moveSeparator already checks if the requested move is allowed
- we already return early when the target is an arrow